src/documentmanager/documentmanager.cpp

```diff
@@ -298,7 +308,7 @@ QMdiSubWindow* DocumentManager::openedDoc(const QString &fileName)
         openedDocIter = std::find_if(subWdwList.cbegin(), subWdwList.cend(),
                                      [&fileName](const auto& doc)
         {
-            return static_cast<CodeEditor*>(doc->widget())->getFileName() == fileName;
+            return qobject_cast<CodeEditor*>(doc->widget())->getFileName() == fileName;
```

Comment:
qobject_cast can return nullptr. after that you're trying to call method getFileName. If you call this fucntion to nullptr it'll crush the program. Check what cast returned before calling the method

Reply:
fixed